packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useMemo, useState } from 'react';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { useModal } from '../../../../../components/ModalProvider';
+import DocumentsIcon from '../../../../../public/images/ic-documents.svg';
+import IdCardIcon from '../../../../../public/images/ic-id-card.svg';
+import SelfieIcon from '../../../../../public/images/ic-selfie.svg';
+import { useDynamicLeverageModalState } from '../../../components/DynamicLeverageContext';
+import { getJurisdictionContents } from '../jurisdiction-contents/jurisdiction-contents';
+import { TJurisdictionCardItems, TJurisdictionCardSection } from '../jurisdiction-contents/props.types';
+import {
+    JurisdictionCardClass,
+    JurisdictionCardClassProps,
+    JurisdictionCardTagClass,
+} from './JurisdictionCard.classnames';
+import JurisdictionCardBack from './JurisdictionCardBack';
+import JurisdictionCardRow from './JurisdictionCardRow';
+import JurisdictionCardTag from './JurisdictionCardTag';
+import JurisdictionCardVerificationTag from './JurisdictionCardVerificationTag';
+
+type TJurisdictionCardProps = JurisdictionCardClassProps & {
+    jurisdiction: string;
+    onSelect: (clickedJurisdiction: string) => void;
+    tag?: string;
+};
+
+type TDisplayTextSkinColor =
+    | 'brown-dark'
+    | 'red-dark'
+    | 'red-darker'
+    | 'red-light'
+    | 'voilet-dark'
+    | 'yellow-dark'
+    | 'yellow-light';
```

Comment:
can we extract the type from `JurisdictionCardTagProps`?

Reply:
No we can not and if we use TJurisdictionCardSectionTitleIndicators it has `displayTextSkinColor?: string`